Week3/moreJs-9.js

```diff
@@ -0,0 +1,20 @@
+/*
+Change the function vehicle to use the list of question 7.
+ So that vehicle("green", 3, 1) prints "a green new bike".
+*/
+
+function vehicle (color, type, age){
+    let vehicles = ['motorbike' , 'caravan', 'bike', 'car'];
```

Comment:
What happens if you pass '10' as value for the type as arguments? Will this still work?

Reply:
it returns like ' undefined ' .
I added an if statement that logs a warning when the value is out of range ( length of array).